lib/msf/ui/console/command_dispatcher/creds.rb

```diff
@@ -471,7 +471,7 @@ def creds_search(*args)
         private_val = core.private ? core.private.to_s : ""
         realm_val = core.realm ? core.realm.value : ""
         human_val = core.private ? core.private.class.model_name.human : ""
-        jtr_val = core.private.jtr_format ? core.private.jtr_format : ""
```

Comment:
This looks like a simple typo. @h00die?

Reply:
Initially a lack of insight that private could be null, or that jtr_format could be null.  As linked in #11433, looks like I resolved that in one place, but not the other!